docs/Advanced-Usage/Manager/Manager-Configuration-Files.rst

```diff
@@ -526,11 +533,18 @@ Indicates where the bitstream (FPGA Image) is located, may be one of:
   * A Uniform Resource Identifier (URI), (see :ref:`uri-path-support` for details)
   * A filesystem path available to the manager. Local paths are relative to the `deploy` folder.
 
-``deploy_quadruplet_override``
-""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+``bit``
```

Comment:
can we call this bitstream instead of just bit

Reply:
I'll make this `bitstream_tar`